packages/agent/src/database/config.ts

```diff
@@ -84,20 +73,12 @@ const postgresConfig: PostgresConnectionOptions = validatePostgresOptions({
   cache: DB_CACHE_ENABLED !== 'false',
   entities: [
     ...VeramoDataStoreEntities,
-    ...DataStoreStatusListEntities,
-    ...DataStoreContactEntities,
-    ...DataStoreEventLoggerEntities,
-    ...DataStoreIssuanceBrandingEntities,
-    ...DataStorePresentationDefinitionEntities,
+    ...DataStoreEntities,
   ],
   migrations: [
     ...VeramoDataStoreMigrations,
-    ...DataStoreStatusListMigrations,
-    ...DataStoreContactMigrations,
+    ...DataStoreMigrations,
```

Comment:
not sure if we want all the datastore migrations here. you are now adding `DataStoreEventLoggerMigrations` and `DataStoreMachineStateMigrations` which were not there before. do we want them here?

Reply:
Awaiting input whether we need
...DataStoreStatusListMigrations,
...DataStoreEventLoggerMigrations,
for the holder role

Reply:
This is the agent, it can also host issuers & verifiers now so it needs all DB tables